js/src/admin/components/ExtensionPage.js

```diff
@@ -126,7 +126,9 @@ export default class ExtensionPage extends Page {
             </Button>
           ) : settings ? (
             <div className="Form">
-              {this.getSettings(settings)}
+              {settings.map((setting) => {
+                return this.getSetting(setting);
```

Comment:
Couldn't we do `settings.map(this.getSetting)`?

Reply:
We can do `settings.map(setting => this.getSetting(setting))`

That would mess with `this` context.

Reply:
Can also do `settings.map(this.getSetting.bind(this))`

Reply:
Done